ref/license-information.md

```diff
@@ -0,0 +1,8 @@
+Anbox Cloud is built using multiple open source software components. This guide lists those components and where to find their license information.
+
+| Software components | Where to find their licenses |
+|--|--|
+| LXD images | Within the LXD image, <br/>`/usr/share/licenses` and `/opt/mesa/share/licenses` |
```

Comment:
If people are interested in checking the copyright file for anbox, check the following path 
/usr/share/doc/anbox/copyright

Reply:
Better use `/usr/share/doc/*/copyright`